src/AssetManager/View/Helper/Asset.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace AssetManager\View\Helper;
+
+use AssetManager\Exception\InvalidArgumentException;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use Zend\View\Helper\AbstractHelper;
+use Zend\Cache\Storage\Adapter\AbstractAdapter as AbstractCacheAdapter;
+
+class Asset extends AbstractHelper
+{
+    /**
+     * @var array
+     */
+    private $config;
+
+    /**
+     * @var ServiceLocatorInterface
+     */
+    private $serviceLocator;
+
+    /**
+     * @param ServiceLocatorInterface $serviceLocator
+     */
+    public function __construct(ServiceLocatorInterface $serviceLocator)
```

Comment:
I can pass the config, but I need the serviceLocator not only for getting the AssetManager, but also for the cache [on line 85 of this file](https://github.com/pensiero/AssetManager/blob/feature/asset-view-helper/src/AssetManager/View/Helper/Asset.php#L85)